samples/openapi3/client/petstore/python/tests/test_model.py

```diff
@@ -78,39 +78,27 @@ def test_equal(self):
         self.assertFalse(self.pet1 == self.pet2)
 
     def test_oneof_schema_2_validator(self):
-        new_color = petstore_api.Color()
         array_of_integers = [12, 34, 56]
 
         try:
-            new_color.oneof_schema_2_validator = array_of_integers
-            self.fail(f"Should have failed: {new_color.oneof_schema_2_validator}")
+            new_color = petstore_api.Color(oneof_schema_2_validator=array_of_integers)
```

Comment:
@multani, Hi! Want to discuss with you my changes here 🙏

You introduced `actual_instance = None` and tests about per-variant validators a few months ago. I changed it back to required and tests are became sassy:
- Now you cannot populate the `oneof_schema_N_validator` via assignment, because it will cause validation error for required `actual_instance`
- `__init__` allows population of `oneof_schema_N_validator` field and validates it, but the field never became `actual_instance` value, so it still fails with `actual_instance is required` even if the value is correct
- So in the old code if you run `color=Color(); color.oneof_schema_2_validator = [1, 2, 3]; color.to_json()` you'll get "null" because actual instance is empty. I don't find issues about it, so probably this feature with variant assignment has no production-wide usage 🤞

So it looks like I have two ways to fix it:
- Add field_validators for `oneof_schema_N_validator` field, so it will also populate the `actual_instance` field. In this case most of the old code will work. It will fix issue with empty actual_instance, so no more validation errors after validating variant field.
- Delete or hide the logic with populating `oneof_schema_N_validator` via assignment and `__init__`, because it's internal part related to `actual_instance` validation and it never worked before as a public API. Maybe hide it with underscore or something like this.

What do you think?